command/server/config.go

```diff
@@ -275,6 +275,9 @@ type TelemetryConfig struct {
 
 	// InfluxDB has the influxdb related settings
 	InfluxDB *InfluxDBConfig `hcl:"influx,block"`
+
+	// Prometheus Address
+	PrometheusAddr string `hcl:"prometheus-addr,optional"`
```

Comment:
Still missing two things here:
- Add the default value for Prometheus in the default config (default="")
- Add the `prometheus-addr` flag in the cli.